frontend/src/utils/helpers.js

```diff
@@ -37,7 +37,6 @@ export const createItemIdsArray = (array, idField) => array.map((item) => item[i
 
 export const createDropDownList = (array, labelField, valueField) =>
   array.map((item) => ({ value: item[valueField], label: item[labelField] }));
-
 // Function to create a hash given an array of values and labels
```

Comment:
Was this an accident? Seems like an odd change. Obviously not a big deal either way

Reply:
Accident.